add-on/chrome/modules/MozLoopService.jsm

```diff
@@ -879,6 +883,19 @@ var MozLoopServiceInternal = {
   },
 
   /**
+   * Pause or resume all chat windows that are open.
+   */
+  pauseAllChatWindows(paused) {
```

Comment:
Can you rename this to `toggleBrowserSharing(on = true)`? At least, that's the best I can come up with...